lte/cloud/go/services/subscriberdb/storage/digest.go

```diff
@@ -0,0 +1,222 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package storage
+
+import (
+	"database/sql"
+
+	"magma/orc8r/cloud/go/clock"
+	"magma/orc8r/cloud/go/sqorc"
+
+	"github.com/Masterminds/squirrel"
+	"github.com/pkg/errors"
+)
+
+type digestLookup struct {
+	db      *sql.DB
+	builder sqorc.StatementBuilder
+}
+
+type digestInfo struct {
+	network         string
+	digest          string
+	lastUpdatedTime int64
+}
+
+type DigestInfoSet []digestInfo
+
+// GetNetworks returns a list of network IDs for all digests in DigestInfoSet.
+func (digestInfoSet DigestInfoSet) GetNetworks() []string {
```

Comment:
Just call this "Networks" 

From https://magma.github.io/magma/docs/next/contributing/contribute_conventions#go --

```
Verbify function names
	- Exception: composable method names with well-understood functionality, e.g. foo.Filter(...).Keys().Sorted()
```